bobtemplates/workspace/+package.fullname+/+package.part_1+/+package.part_2+/tests/test_workflows.py.bob

```diff
@@ -2,34 +2,83 @@ from ftw.lawgiver.tests.base import WorkflowTest
 from {{{package.fullname}}}.testing import {{{package.part_1_upper}}}_FUNCTIONAL
 from path import Path
 import ftw.workspace
+import os
 
 
-class CopyFromWorkspaceWorkflowTest(WorkflowTest):
+class CompareWorkspaceWorkflowTest(WorkflowTest):
     layer = {{{package.part_1_upper}}}_FUNCTIONAL
 
+    specification_name = 'specification.de.txt'
+
     def setUp(self):
-        super(CopyFromWorkspaceWorkflowTest, self).setUp()
-        if not self._is_base_test():
-            spec_source = Path(ftw.workspace.__file__).joinpath(
-                '..', 'profiles', 'default',
-                'workflows', Path(self.workflow_path).basename(),
-                'specification.de.txt').abspath()
-            spec_target = Path(__file__).joinpath(
-                '..', self.workflow_path).abspath()
-            spec_source.copy(spec_target)
+        super(CompareWorkspaceWorkflowTest, self).setUp()
+        if self._is_base_test():
+            return
+
+        spec_source_path = Path(ftw.workspace.__file__).joinpath(
+            '..', 'profiles', 'default',
+            'workflows', Path(self.workflow_path).basename(),
+            self.specification_name).abspath()
+
+        self.assertTrue(
+            os.path.isfile(spec_source_path),
```

Comment:
The Path-object has methods for such things:
```
spec_source_path.isfile()
```
See http://pythonhosted.org/path.py/api.html#path.Path.isfile 😉